todo_app/data/item_lists.py

```diff
@@ -13,7 +13,7 @@ def name(self) -> str:
         return self._name
     
     @property
-    def items(self) -> List[Item]:
+    def items(self) -> List[MongoItem]:
```

Comment:
Should the user care that this happens to be stored with Mongo? I'd argue that this is a good example of where changing the class name is a mistake - this represents an `Item` in our world, regardless of where it's stored, and this ViewModel doesn't really care where the underlying data was